removestar/removestar.py

```diff
@@ -112,14 +112,16 @@ def replace_imports(code, repls, *, max_line_length=100, file=None, verbose=Fals
                                   name3)
 
     """
+    warning_prefix = f"Warning: {file}: " if file else "Warning: "
+    verbose_prefix = f"{file}: " if file else ""
+
     for mod in repls:
         names = sorted(repls[mod])
 
-        STAR_IMPORT = re.compile(rf'from +{re.escape(mod)} +import +\*\n')
         if not names:
             new_import = ""
         else:
-            new_import = f"from {mod} import " + ', '.join(names) + '\n'
+            new_import = f"from {mod} import " + ', '.join(names)
```

Comment:
Oh, I think the line wrapping problem is here. I have some code that was exactly 99 characters without the newline. I think the line below needs to be adjusted to `len(new_import) + 1 - len(names[-1]) > max_line_length`. 

If I understand the logic correctly, there's really +2 here, one for the comma and one for the space. One is already accounted from the fact that a wrapped line gets an extra `(` added, and one was accounted for before from the extra `\n` in `new_import`.

Reply:
There is a test for this, `test_replace_imports_line_wrapping`, but I guess it doesn't test the edge between no wrapping at all and wrapping into two lines, which is where this logic is relevant because that's where we add the `(`.

Reply:
Actually, I think it should just be `len(new_import) > max_line_length`. I'm not sure why I was removing the last name when checking this. That makes it only line wrap if the line is too long not counting the last name.

Reply:
Fixed in https://github.com/asmeurer/removestar/pull/24